packages/shop/src/hooks/use-search/useSearch.ts

```diff
@@ -16,11 +16,13 @@ type Options = {
 export const useSearch = (options: Options) => {
   const { searchWord, staleTime } = options;
 
+  const trimmedSearchWord = searchWord.trim();
```

Comment:
nit: I think it'd be good to consider text normalize.

Reply:
The search API will be responsible for normalization.